src/components/tearComponent/index.stories.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { select, text, number } from '@storybook/addon-knobs'
+import sampleBackground from './fixture/Background.png'
```

Comment:
Can you use the logo in `src/assets/logo.png`? Trying to keep some consistency across stories and reduce the codebase size 🙂